src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -164,34 +164,37 @@ function getRouteForCurrentStep(currentStep) {
     }
 }
 
-function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, policy, account, isLoadingApp, session, plaidLinkToken, plaidCurrentEvent, reimbursementAccountDraft}) {
+function ReimbursementAccountPage({
+    reimbursementAccount,
+    route,
+    onfidoToken = '',
+    policy,
+    account,
+    isLoadingApp = false,
+    session,
+    plaidLinkToken = '',
+    plaidCurrentEvent = '',
+}: ReimbursementAccountPageProps) {
     /**
         The SetupWithdrawalAccount flow allows us to continue the flow from various points depending on where the
         user left off. This view will refer to the achData as the single source of truth to determine which route to
         display. We can also specify a specific route to navigate to via route params when the component first
         mounts which will set the achData.currentStep after the account data is fetched and overwrite the logical
         next step.
     */
-    const achData = lodashGet(reimbursementAccount, 'achData', {});
+    const achData = reimbursementAccount?.achData;
 
-    /**
-     * @param {Array} fieldNames
-     *
-     * @returns {Object}
-     */
-    function getBankAccountFields(fieldNames) {
+    function getBankAccountFields(fieldNames: string[]): string[] {
+        // @ts-expect-error -- Pick<ACHData, T> is more acurate type in this case because lodashPick returns Partial<ACHData>
```

Comment:
Then why we have `string[]` as a return type?

Reply:
lol I dont know what happend here 😅